src/transport/SecureSessionMgr.cpp

```diff
@@ -384,24 +384,27 @@ void SecureSessionMgr::SecureMessageDispatch(const PacketHeader & packetHeader,
     }
 
     admin = mAdmins->FindAdminWithId(state->GetAdminId());
-    VerifyOrExit(admin != nullptr,
-                 ChipLogError(Inet, "Secure transport received packet for unknown admin (%p, %d) pairing, discarding", state,
-                              state->GetAdminId()));
+    VerifyOrExit(
+        admin != nullptr,
+        ChipLogError(Inet, "Secure transport discarding message, unknown admin (%p, %u) pairing", state, state->GetAdminId()));
     if (packetHeader.GetDestinationNodeId().HasValue() && admin->GetNodeId() != kUndefinedNodeId)
     {
-        VerifyOrExit(
-            admin->GetNodeId() == packetHeader.GetDestinationNodeId().Value(),
-            ChipLogError(
-                Inet,
-                "Secure transport received message, but destination node ID (%llu) doesn't match our node ID (%llu), discarding",
-                packetHeader.GetDestinationNodeId().Value(), admin->GetNodeId()));
+        VerifyOrExit(admin->GetNodeId() == packetHeader.GetDestinationNodeId().Value(),
+                     ChipLogError(Inet,
+                                  "Secure transport discarding message, destination node ID (0x%08" PRIx32 "%08" PRIx32
+                                  ") <> our node ID (0x%08" PRIx32 "%08" PRIx32 ")",
+                                  static_cast<uint32_t>(packetHeader.GetDestinationNodeId().Value() >> 32),
```

Comment:
I understand the change is needed for ARM nano stdlib, but I wonder if adding the following helper macros wouldn't result in more readable code.
```
#define ChipLogFormatX64 "0x%08" PRIx32 "%08" PRIx32
#define ChipLogValueX64(aValue) static_cast<uint32_t>(aValue >> 32), static_cast<uint32_t>(aValue)
printf(ChipLogFormatX64, ChipLogValueX64(packetHeader.GetDestinationNodeId().Value()));
```

Reply:
It would indeed - what would be the best place to put such a macro ?

Reply:
`CHIPLogging.h`?